action.yml

```diff
@@ -4,17 +4,21 @@
   GITHUB_TOKEN:
     description: "GitHub token to interact with the repository."
     required: true
-  OPENAI_API_KEY:
-    description: "OpenAI API key for GPT."
+  INFERENCE_API_KEY:
+    description: "OpenAI or Anthropic API key for LLM inference."
     required: true
-  OPENAI_API_MODEL:
-    description: "OpenAI API model."
+  MODEL:
```

Comment:
The change from `OPENAI_API_MODEL` to `MODEL` is vague. Ensure that the new name `MODEL` is sufficiently descriptive for users to understand what models are supported.